<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="0" skipped="0" tests="1" time="0.223" timestamp="2023-11-29T01:33:23.877312" hostname="a1f8948d47fc"><testcase classname="" name="tests.test_pytest_cov" time="0.000"><error message="collection failure">ImportError while importing test module '/workdir/pytest-cov-4.0.0/tests/test_pytest_cov.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_pytest_cov.py:14: in &lt;module&gt;
    import xdist
E   ModuleNotFoundError: No module named 'xdist'</error></testcase></testsuite></testsuites>